clusterloader2/pkg/measurement/common/etcd_metrics.go

```diff
@@ -43,15 +43,15 @@ func createEtcdMetricsMeasurement() measurement.Measurement {
 	return &etcdMetricsMeasurement{
 		stopCh:  make(chan struct{}),
 		wg:      &sync.WaitGroup{},
-		metrics: newEtcdMetrics(),
+		metrics: make(map[string]*etcdMetrics),
 	}
 }
 
 type etcdMetricsMeasurement struct {
 	isRunning bool
 	stopCh    chan struct{}
 	wg        *sync.WaitGroup
-	metrics   *etcdMetrics
+	metrics   map[string]*etcdMetrics
```

Comment:
The intent here is to organize the etcdMetrics per host (master IP) in order to have `e.startCollecting()` and `e.stopAndSummarize()` collect data on a single host.
The keys for the map are strings in order to conform with [`ClusterConfig.MasterIPs`](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/config/cluster.go#L37).